arviz/plots/essplot.py

```diff
@@ -164,7 +164,7 @@ def plot_ess(
             dim="ess_dim",
         )
     elif kind == "local":
-        probs = np.arange(0, 1, 1 / n_points)
+        probs = np.linspace(0, 1 - 1 / n_points, n_points)
```

Comment:
Is this same as 

    np.linspace(0,1,N, endpoint=False)

(could be different)